game/die.h

```diff
@@ -0,0 +1,13 @@
+#include <stdlib.h>     /* srand, rand */
+#include <time.h> 
+
+// Class to represent a die of n sides.
+class Die {
+public:
+    Die(int n) {
+        sides = n;
```

Comment:
You should use an initializer list here instead.

Reply:
done